components/layout_2020/positioned.rs

```diff
@@ -25,7 +25,14 @@ pub(crate) struct AbsolutelyPositionedBox {
 
 pub(crate) struct PositioningContext<'box_tree> {
     for_nearest_positioned_ancestor: Option<Vec<HoistedAbsolutelyPositionedBox<'box_tree>>>,
-    for_initial_containing_block: Vec<HoistedAbsolutelyPositionedBox<'box_tree>>,
+
+    // For nearest `containing block for all descendants` as defined by the CSS transforms
+    // spec.
+    // https://www.w3.org/TR/css-transforms-1/#containing-block-for-all-descendants
+    //
+    // We use the abbreviation `cbfad` here only because `containing block for all descendants`
+    // is so unwieldy.
```

Comment:
😆

Reply:
Personally I think `containing_block_for_all_descendants` is way better, and not even the longest identifier in layout 2020.

Reply:
We have a `next_in_flow_margin_collapses_with_parent_start_margin`.

Reply:
Okay. I've changed this to be `containing_block_for_all_descendants` everywhere.